src/main/java/reposense/report/CommitReportJson.java

```diff
@@ -21,28 +19,6 @@
     private final Map<Author, Float> authorContributionVariance;
     private final Map<Author, String> authorDisplayNameMap;
 
-    /**
```

Comment:
This will conflict with #777,  can we have a discussion on how to deal with no author error?

Reply:
I think repos (with no preset author in config files) that has no commits on the specified dates (--> no authors found) should not be an error, but rather have a placeholder stating that `no commits were found during this time period`.